assign-2/src/Presentation/Main.java

```diff
@@ -0,0 +1,90 @@
+package Presentation;
+
+import data.StudentDao;
+import data.ArrayListStudentDao;
+import logic.Student;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+
+public class Main {s
+    public static final String ADD_MENU_ITEM = "1";
```

Comment:
You are right,since I am using those constants only inside the Main class, it would be better to make them private instead of public. This will help to encapsulate the data and prevent other classes from accessing these constants unnecessarily. I will make the necessary changes and resubmit the code for review.